### PR TITLE
Biancalina reviews

### PR DESCRIPTION
**Pull request by biancalina**

### CODE REVIEW

File: src/_component/portfolio/portfolio.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.portfolio__section {
+  @extend %align-items;
+  position: relative;
+}
+
+.portfolio-first__title,
+.portfolio-second__title {
+  color: #707070;
```
Comment (pavelciupitu) [inline src/_component/portfolio/portfolio.scss]: All colors should be declared in a sass file - (name that color - Dove Gray)

Comment (pavelciupitu) [reply src/_component/portfolio/portfolio.scss]: So every time you re-use the color #707070 
color: $color-Dove-Gray

File: src/_component/portfolio/portfolio.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+.codetap-content,
+.pug-content {
+  display: inline-block;
+  text-align: initial;
+  padding: 15px;
+}
+
+.codetap-title,
+.pug-title {
+  color: #808080;
```
Comment (pavelciupitu) [inline src/_component/portfolio/portfolio.scss]: And also this one. :)

File: src/_component/recommendation/recommendation.pug [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    .recommendation-wrapper
+        .recommendation-user
+            span.my-cv-user
+        .recommendation-detail
+            h1.recommendation-name Simona Radulescu
+            h2.recommendation-job Software Engineer - JavaScript
+    .space
+    .recommendation-box
+        p.recommendation-text Bianca is a highly creative and easy-going person to work with. He is multi-skilled, has a vast knowledge in web development in general and frontend development in particular with jaw-dropping CSS skills. In addition, he is proactive, self motivated, intelligent, keen to learn new technologies, always updating his knowledge/skills and keen to share his vast knowledge with any and all. It was a pleasure working with him. Bianca makes a great addition to any team!
```
Comment (pavelciupitu) [inline src/_component/recommendation/recommendation.pug]: Re-read the text recommendation - after the first sentence, it's about a boy.

File: src/_component/portfolio/portfolio.scss (hunk 1 above)
Comment (biancalina) [reply src/_component/portfolio/portfolio.scss]: Yes, you are right! I forgot to do it but I am going to change it now. Those pictures are only temporary, soon I will change everything with real content. Thank you for your review!

File: src/_component/recommendation/recommendation.pug (hunk 3 above)
Comment (biancalina) [reply src/_component/recommendation/recommendation.pug]: I changed. Thank you!

File: src/_component/portfolio/portfolio.scss (hunk 2 above)
Comment (biancalina) [reply src/_component/portfolio/portfolio.scss]: Done!

**Pull request merged**